server/plugin/plugin_test.go

```diff
@@ -21,7 +21,7 @@ import (
 	"golang.org/x/text/language"
 )
 
-func setupTestPlugin(t *testing.T, api *plugintest.API, store *mockstore.Store) *MatterpollPlugin {
+func setupTestPlugin(_ *testing.T, api plugin.API, store store.Store) *MatterpollPlugin {
```

Comment:
Why `plugintest.API` and `mockstore.Store` are replaced with `plugin.API` and  `store.Store`?

Reply:
Because `interfacer` suggested this :man_shrugging:

You are right, this is confusing. I will revert.